### PR TITLE
Add grafana to dev mimir-monolithic-mode and mimir-read-write-mode docker composes

### PR DESCRIPTION
**Pull request by jhalterman**
#### What this PR does

This PR adds grafana to the dev mimir-monolithic-mode and mimir-read-write-mode docker compose setups, pre-configuring it with a datasource for the query-frontend.

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: development/mimir-monolithic-mode/config/datasource-mimir.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: 1
+datasources:
+- name: Mimir
```
Comment (charleskorn) [inline development/mimir-monolithic-mode/config/datasource-mimir.yaml:3]: Should we make this datasource the default? (so set `isDefault: true`)

File: development/mimir-read-write-mode/config/datasource-mimir.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+datasources:
+- name: Mimir
```
Comment (charleskorn) [inline development/mimir-read-write-mode/config/datasource-mimir.yaml:3]: Same as above: should we make this datasource the default?

**Review by charleskorn — COMMENTED** [review]
LGTM, apart from suggestion to make the data sources the default.

Should we also add a Mimir datasource to the microservices setup as well?

**Review by jhesketh — APPROVED** [review]
lgtm :-)

File: development/mimir-monolithic-mode/config/datasource-mimir.yaml (hunk 1 above)
Comment (jhalterman) [reply development/mimir-monolithic-mode/config/datasource-mimir.yaml:3]: Good idea - I'll also make the mimir datasource default in the microservices docker-compose, for consistency.

**Review by charleskorn — APPROVED** [review]

**Pull request merged**